pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/UnnecessaryConstructorRule.java

```diff
@@ -35,7 +35,8 @@ public class UnnecessaryConstructorRule extends AbstractIgnoredAnnotationRule {
 
     @Override
     protected Collection<String> defaultSuppressionAnnotations() {
-        return Collections.singletonList("javax.inject.Inject");
+        return Arrays.asList("javax.inject.Inject",
+                "com.google.inject.Inject");
```

Comment:
we should probably also include Spring's `@Autowired` here…

Reply:
Thanks for comment. I will add Spring's @Autowired later